lib/roadie/document.rb

```diff
@@ -29,12 +29,24 @@ class Document
     # Should CSS that cannot be inlined be kept in a new `<style>` element in `<head>`?
     attr_accessor :keep_uninlinable_css
 
+    # Should non-inlineable media queries be grouped?
+    # e.g.
+    # @media(max-width: 600px) { .col-6 { display: block; } }
+    # @media(max-width: 400px) { .col-12 { display: inline-block; } }
+    # @media(max-width: 600px) { .col-12 { display: block; } }
+    # will become
+    # @media(max-width: 600px) { .col-6 { display: block; } .col-12 { display: block; } }
+    # @media(max-width: 400px) { .col-12 { display: inline-block; } }
+    # which would change the styling on the page
```

Comment:
I don't think this will render correctly in Yardoc. Lines beginning with `@` are Yard macros.
Please look at other methods to see how you can embed example code.

---

I think the first sentence should not be a question. What do you think about this?

> Merge media queries to increase performance and reduce email size if enabled.
>
> This will change specificity in some cases, like for example:

Reply:
Yep @Mange - codeblocks need to be treated a little different. An additional two spaces before `@media...` will suffice.